baseplate/clients/redis.py

```diff
@@ -3,7 +3,13 @@
 from typing import Dict
 from typing import Optional
 
-import redis.client
+import redis
+
+# redis.client.StrictPipeline was renamed to redis.client.Pipeline in version 3.0
+try:
+    from redis.client import StrictPipeline as Pipeline
+except ImportError:
+    from redis.client import Pipeline  # type: ignore
```

Comment:
I feel bad adding this `type: ignore`, but I was unable to find a better solution. Open issue in mypy:
https://github.com/python/mypy/issues/1153

Reply:
seems reasonable, can't please the linter all the time.